Sources/Models/FaTsaiBrain.swift

```diff
@@ -33,80 +33,98 @@ final class FaTsaiBrain {
     init(totalNumbersCount: Int, numbersRange: (first: Int, last: Int)) {
         self.totalNumbersCount = totalNumbersCount
         self.numbersRange = numbersRange
-    }    
+    }
+    
+    func randomNumbers() -> [Int] {
+        
+        guard requiredNumbersInRange() else {
+            return []
+        }
+        guard numbersInRange(inclusiveNumbers, numbersRange: (numbersRange.first, numbersRange.last)) else {
```

Comment:
Function is unclear about what it does.  Maybe we can call something like `isInclusiveNumbersInRange`?

Reply:
Do we need the second parameter?